chapter4/tests/t16/.nginx.conf.d/camp-php.local-server.conf

```diff
@@ -0,0 +1,18 @@
+server_name camp-php.local;
+
+root /workspace/htdocs;
+
+location /reports/fast {
+  add_header x-server-engine "php";
+  include fastcgi_params;
+  fastcgi_param SCRIPT_FILENAME $realpath_root/fast.php;
+  fastcgi_pass php-www;
+}
+
+location /reports/slow {
+  add_header x-server-engine "php";
+  include fastcgi_params;
+  fastcgi_param SCRIPT_FILENAME $realpath_root/slow.php;
+  fastcgi_read_timeout 11m;
+  fastcgi_pass php-www-slow;
+}
```

Comment:
that should be wrapped in the server block context

Reply:
It is already in "server" block: https://github.com/paketo-buildpacks/php-nginx/blob/main/assets/nginx.conf . If I wrap in "server" it will be nested block.

Reply:
@dmitry-mightydevops pls let me know if I'm wrong.